config/settings/base.py

```diff
@@ -30,16 +30,16 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-)4-5u$x8-hcosa*l6w!*9e*&x$lwh5s+tmd)ve_(ft+v04gvhn"
+SECRET_KEY = os.getenv("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = bool(os.environ.get("DEBUG", default=0))
```

Comment:
```suggestion
DEBUG = False
```

Et inclure un `settings/dev.py` qui fait :
```
from .base import *

DEBUG = True
```

Comme ça, DEBUG vaudra toujours `False` en prod.